components/avatar/avatar.vue

```diff
@@ -3,20 +3,34 @@
     :id="id"
     :class="[
       'd-avatar',
-      AVATAR_KIND_MODIFIERS[kind],
       AVATAR_SIZE_MODIFIERS[size],
       avatarClass,
+      gradient ? null : 'd-avatar--no-gradient',
+      showGroup ? 'd-avatar--group' : null,
```

Comment:
use object syntax rather than a ternary here

```
:class="[
  'd-avatar',
  AVATAR_SIZE_MODIFIERS[size],
  avatarClass,
  {
    'd-avatar--no-gradient': gradient,
    'd-avatar--group': showGroup,
  },
]"
```

Reply:
Good call, refactored and moved to a computed prop as that's the convention we've been using https://github.com/dialpad/dialtone-vue/pull/745/commits/3cd94c9122b5ec6454d465245239f14248354624